forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -157,7 +153,7 @@ export default function CreateNewFilterDrawer({
       if (!selectedFilterData.variables?.length) {
         setInputValues([{ name: "", label: "" }]);
       }
-      if(selectedFilterData.variables?.length === 2){
+      if (selectedFilterData.variables?.length === 2) {
```

Comment:
the above if the above "IF" condition true it will execute same instruction right so you can add a or condition like this:
 `  if (!selectedFilterData.variables?.length || selectedFilterData.variables?.length === 2 ) {
        setInputValues([{ name: "", label: "" }]);
      }`